solidity/contracts/BondedECDSAKeep.sol

```diff
@@ -527,127 +78,48 @@ contract BondedECDSAKeep is IBondedECDSAKeep {
         address _keepBonding,
         address payable _keepFactory
     ) public {
-        require(!isInitialized, "Contract already initialized");
+        initializeBondedECDSAKeep(
+            _owner,
+            _members,
+            _honestThreshold,
+            _keepBonding
+        );
 
-        owner = _owner;
-        members = _members;
-        honestThreshold = _honestThreshold;
         memberStake = _memberStake;
         tokenStaking = TokenStaking(_tokenStaking);
-        keepBonding = KeepBonding(_keepBonding);
         keepFactory = BondedECDSAKeepFactory(_keepFactory);
```

Comment:
We don't need `keepFactory` field, we can delete this line. All we need the `keepFactory` for here is to `claimDelegatedAuthority`. It's nothing new, we should clean this up earlier but now we have the opportunity.

Reply:
Yes, I was thinking about it but wasn't 100% sure about removing. Glad you mentioned it. I'll do that.